quickstart/101-vm-with-infrastructure/variables.tf

```diff
@@ -8,4 +8,11 @@ variable "resource_group_name_prefix" {
   type        = string
   default     = "rg"
   description = "Prefix of the resource group name that's combined with a random ID so name is unique in your Azure subscription."
+}
+
+
+variable "username" {
```

Comment:
Since we've set default value for this variable I think we can set `nullable = false` for this block too.